index.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import db from "./src/config/dbConfig";
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+
+app.listen(PORT, console.log(`Listening on port ${PORT}.......`));
+
+db.authenticate()
```

Comment:
Shouldn't the application be started after a database connection has been established?

Reply:
Yeah, sorry I didn't think about this because I thought it was only for testing. let me change that too.thanks